api/Element.json

```
@@ -7076,7 +7076,8 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "8",
+              "notes": "The value of <code>scrollWidth</code> affected by padding value from Internet Explorer 8 through 9."
```

I think we can word this better:

```suggestion
              "notes": "In Internet Explorer 5 through 7, if padding is set, the value of <code>scrollWidth</code> is equal to the sum of the left and right padding.  This behavior was fixed in Internet Explorer 8."
```

---

Have you specified the DOCTYPE, I did not specify it in my test

---

I pretty much never do when performing tests; it’s kind of like adding a flag to the top of a plain text file saying it’s text, when it’s already pretty obvious.  😛